README.md

```diff
@@ -1 +1,9 @@
-# cheerybin.desktop
\ No newline at end of file
+# cheerybin.desktop
+
+## Development
+
+```sh
+yarn
```

Comment:
I propose switching back to npm. For [speed](https://blog.npmjs.org/post/171556855892/introducing-npm-ci-for-faster-more-reliable) and [security](https://blog.npmjs.org/post/173719309445/npm-audit-identify-and-fix-insecure).